storage_controller/src/persistence.rs

```diff
@@ -1418,10 +1446,11 @@ impl Persistence {
         let tenant_id = &tenant_id;
         let timeline_id = &timeline_id;
         self.with_measured_conn(DatabaseOperation::RemoveTimelineReconcile, move |conn| {
+            let timeline_id_str = timeline_id.map(|tid| tid.to_string()).unwrap_or_default();
```

Comment:
Having null in the db would be cleaner.

Reply:
apparently it can't be made nullable any more:
```sql
ALTER TABLE safekeeper_timeline_pending_ops ALTER COLUMN timeline_id DROP NOT NULL;
```
gives:
```
Error: Migration error: Failed to run 2025-03-19-142550_safekeeper_op_nullable_fields with: column "timeline_id" is in a primary key
```

Reply:
Indeed, forgot about that. We can make the key unique instead of primary, or just give up on that.